### DISCUSSION TITLE
M30  GamepadAlt mode in TURBO EVERDRIVE PRO menu doesn't work.

### DISCUSSION THREAD

**Discussion by LusterWong** [darthcloud/BlueRetro#887] [Q&A]
My gamepad is 8-BIT Do M30 black.
Did not copy ROM to TURBO EVERDRIVE PRO, just in the menu test. Setting 2-key mode can be operated, but setting 6-key mode has no response, even the D-PAD cannot be operated.
I heard that the 6-button mode works in specific games, but the game won't work if it can't be operated from the menu.

**Comment by darthcloud** [comment]
6 button mode will not work in the menu. The menu support only 2 button.

You can press the home button to toggle blueretro between 2btn and 6 btn mode

**Reply by LusterWong** [reply]
Thank you. I didn't know the HOME button could be used to switch key mode.
